### PR TITLE
New changes to be added

### PR DESCRIPTION
**Pull request by mminhaz93**
- Remove unwanted files
- Update read me
- Update gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,4 +76,10 @@ fabric.properties
 
 .DS_Store
 
-logs/
\ No newline at end of file
+logs/
+
+.env
```
Comment (mminhaz93) [inline .gitignore:81]: Looks good

Comment (jarinmahmud) [inline .gitignore:81]: looks good

File: .gitignore [hunk 2]
```diff
@@ -76,4 +76,10 @@ fabric.properties
...
+
+.password
```
Comment (tanveerhussain58) [inline .gitignore:83]: looks good thank you

File: .gitignore [hunk 3]
```diff
@@ -76,4 +76,10 @@ fabric.properties
 
 .DS_Store
 
-logs/
```
Comment (atikarahman) [inline .gitignore:79]: Looks fine.

File: .gitignore (hunk 2 above)
Comment (mminhaz93) [reply .gitignore:83]: Changes are fixed

**Review by tanveerhussain58 — APPROVED** [review]
thank you welcome

File: .gitignore (hunk 2 above)
Comment (atikarahman) [inline .gitignore:83]: Review

**Review by atikarahman — APPROVED** [review]

**Review by tanveerhussain58 — APPROVED** [review]
thank you welcome

File: .gitignore [hunk 4]
```diff
@@ -76,4 +76,10 @@ fabric.properties
...
\ No newline at end of file
+logs/
+
+.env
+
+.password
+
```
Comment (jarinmahmud) [inline .gitignore:84]: review

**Review by jarinmahmud — APPROVED** [review]

**Pull request merged**